crates/bitwarden/src/tool/generators/username_forwarders/firefox.rs

```diff
@@ -77,6 +78,19 @@ mod tests {
                     "full_address": "ofuj4d4qw@mozmail.com"
                 })))
                 .expect(1),
+            // Mock a missing generated_for field
+            Mock::given(matchers::path("/api/v1/relayaddresses/"))
+                .and(matchers::method("POST"))
+                .and(matchers::header("Content-Type", "application/json"))
+                .and(matchers::header("Authorization", "Token MY_OTHER_TOKEN"))
+                .and(matchers::body_json(json!({
+                    "enabled": true,
+                    "description": "Generated by Bitwarden."
+                })))
+                .respond_with(ResponseTemplate::new(201).set_body_json(json!({
+                    "full_address": "856f7765@mozmail.com"
+                })))
+                .expect(1),
```

Comment:
It's generally easier to debug if each case is it's own separate test, vs trying to figure out what belongs to what.

Reply:
That's a solid point, I've split them into separate test functions now.